gef-ui/server/utils/helpers.js

```diff
@@ -228,21 +228,27 @@ const stringToBoolean = (str) => (str === 'false' ? false : !!str);
 
 const isNotice = (type) => type.toLowerCase().includes('notice');
 
-const isUkefReviewAvailable = (applicationStatus) => {
-  const acceptable = [
-    CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITHOUT_CONDITIONS,
-    CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITH_CONDITIONS,
-    CONSTANTS.DEAL_STATUS.UKEF_REFUSED,
-  ];
-  return acceptable.includes(applicationStatus);
+const isUkefReviewAvailable = (application) => {
+  if (application.ukefDecision?.length > 0) {
+    const acceptable = [
+      CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITHOUT_CONDITIONS,
+      CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITH_CONDITIONS,
+      CONSTANTS.DEAL_STATUS.UKEF_REFUSED,
+    ];
+    return acceptable.includes(application.status) || acceptable.includes(application.ukefDecision[0].decision);
+  }
+  return false;
 };
 
-const isUkefReviewPositive = (applicationStatus) => {
-  const acceptable = [
-    CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITHOUT_CONDITIONS,
-    CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITH_CONDITIONS,
-  ];
-  return acceptable.includes(applicationStatus);
+const isUkefReviewPositive = (application) => {
+  if (application.ukefDecision?.length > 0) {
+    const acceptable = [
+      CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITHOUT_CONDITIONS,
+      CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITH_CONDITIONS,
+    ];
+    return acceptable.includes(application.status) || acceptable.includes(application.ukefDecision[0].decision);
+  }
+  return false;
```

Comment:
Do you think we simplify the params passed to these functions, so it only get what it needs? i.e

```js
const isUkefReviewPositive = (ukefDecision, dealStatus) => ....
```